### ISSUE TITLE
Query about Captured Pieces being updated

### ISSUE THREAD

**Issue by zotz** [lyudmil-mitev/Simple-Python-Chess#4]
I am trying to work with the codebase.

In board.py:    we set: captured_pieces = { 'white': [], 'black': [] }

But we never seem to update this as pieces are captured.

Should this be updated somewhere in:

    def _finish_move(self, piece, dest, p1, p2):
    
or elsewhere?

**Comment by lyudmil-mitev** [comment]
Yes, you are correct, the `captured_pieces` property was not used and `_finish_move` could be [modified somewhere here](https://github.com/liudmil-mitev/Simple-Python-Chess/blob/87b41135d30a60c154b8a6d51bca240cde96190c/chesslib/board.py#LL127C15-L127C15) in order to update the captured pieces.
